helix-view/src/editor.rs

```diff
@@ -162,6 +164,10 @@ pub struct Config {
     pub rulers: Vec<u16>,
     #[serde(default)]
     pub whitespace: WhitespaceConfig,
+    /// Vector of Strings of custom spinner frames, one per element
+    pub spinner_frames: Vec<String>,
```

Comment:
Both of these should be nested in an object: `spinner = { frames =, interval =}`